packages/workspace/src/generators/move/lib/update-project-root-files.ts

```diff
@@ -51,7 +51,7 @@ export function updateFilesForRootProjects(
     if (!allowedExt.includes(ext)) {
       continue;
     }
-    if (file === '.eslintrc.json' || file === 'eslint.config.js') {
+    if (file === '.eslintrc.json' || file === 'eslint.config.cjs') {
```

Comment:
Why was this one a replacement when others were applied in addition to the existing?

Reply:
Fixed